docs/reference/settings/monitoring-settings.asciidoc

```diff
@@ -339,35 +339,35 @@ When using Java keystore files (JKS), which contain the private key, certificate
 and certificates that should be trusted, use the following settings:
 
 +{ssl-prefix}.ssl.keystore.path+::
-(<<static-cluster-setting,Static>>) deprecated:[7.16.0] 
+(<<static-cluster-setting,Static>>) deprecated:[7.16.0]
 include::{es-repo-dir}/settings/common-defs.asciidoc[tag=ssl-keystore-path]
 
 +{ssl-prefix}.ssl.keystore.password+::
-(<<static-cluster-setting,Static>>) deprecated:[7.16.0] 
```

Comment:
I don't have a better suggestion, but I think it might be confusing to readers to have:

> `keystore.password` Deprecated 7.17 - use `keystore.secure_password`
>
> `keystore.secure_password` Deprecated 7.16
 
Both are deprecated, and the one you should use (if you're still using stack monitoring) is the one that was deprecated earlier.
The best idea I have is to put some extra words on the `secure_` version to say _Even though this is deprecated you should still use it in preference to the non-secure version_.